### ISSUE TITLE
Insufficient contrast on "Compare" button

### ISSUE THREAD

**Issue by grugnog** [CivicActions/nebula#75]
The contrast on the "compare" button with the white text is too low for accessibility. I tried darkening the background from #3FA9F5  to #0B7ACA , but it ends up basically the same as the surround. Let me know what you would like to do - we could go with black text for that button, or we could make the button darker than the surround instead.

**Comment by ghost** [comment]
Yes, let's change the compare button background color to #404C54. Also, remove the blue border and change font to uppercase. @davenuman is this for you?

**Comment by ghost** [comment]
Reference image
![qa-sideeeffect-hero-btn](https://cloud.githubusercontent.com/assets/13037646/8420820/e7199bc4-1e7b-11e5-985f-6dc6f3173f35.jpg)
